java/ql/src/Likely Bugs/Likely Typos/SuspiciousDateFormat.qhelp

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+
+<overview>
+<p>
+Some <code>SimpleDateFormat</code> patterns might not work correctly at the end of the calendar
+year, due to use of the <code>Y</code> placeholder (representing the ISO 8601 week year), rather
+than <code>y</code> representing the actual year.
+</p>
```

Comment:
Possibly it's worth expanding this overview. Something more like:

The Java <code>SimpleDateFormat</code> class provides many placeholders so that you can define precisely the date format required. However, this also makes it easy to define a pattern that doesn't behave exactly as you intended. The most common mistake is to use the <code>Y</code> placeholder (which represents the ISO 8601 week year), rather than <code>y</code> (which represents the actual year). In this case, the date reported will appear correct until the end of the year, when the "week year" may differ from the actual year.